cgmes/cgmes-model/src/main/java/com/powsybl/cgmes/model/CgmesTerminal.java

```diff
@@ -42,7 +42,7 @@ public CgmesTerminal(PropertyBag t) {
         // two relationships may be different
         // (TODO(Luma): missing reference here)
 
-        this.connected = t.asBoolean("connected", false);
+        this.connected = t.asBoolean("connected", true);
```

Comment:
I'm not sure about that... What do you think @annetill? Should all terminals be considered connected or disconnected? Or should it be configurable?

Reply:
This parameter is obtained from SSH, so if we leave the previous default value, when a model that only has EQ is imported, the entire model will be disconnected.

Reply:
Yes, I understood that and indeed, you may be right, it might be more "logical" to consider all of them connected. Just not sure it is always what we want, depending on the use case and this is why I asked Anne.

Reply:
I discussed with Anne and for now, we think that as you coded, the default value should be to true. However (maybe in another PR), there should be an option to create a switch (instead of internal connections) in these case so users can disconnect manually some terminals afterwards.
I am still marking this conversation as resolved as it will not be coded in this PR.